sites/fast-website/src/app/components/content-card/content-card.styles.ts

```diff
@@ -0,0 +1,51 @@
+import { css } from "@microsoft/fast-element";
+import { display } from "@microsoft/fast-components";
+
+export const ContentCardStyles = css`
+    ${display("block")} :host {
+        contain: content;
+        font-family: var(--body-font);
+        box-sizing: border-box;
+    }
+
+    :host([hidden]) {
+        display: none;
+    }
+
+    fast-card {
+        background: transparent;
+        padding: 0 calc(var(--design-unit) * 5px) calc(var(--design-unit) * 3px);
+        text-align: left;
+    }
+
+    fast-card:hover .contentCard_footer__hoverVisible {
```

Comment:
I don't recall seeing BEM anywhere else. I'm not opposed to it, but just want to make sure we're being consistent. /cc @chrisdholt

Reply:
BEM was discussed in an earlier PR but I don't think it's a really high priority. It's being tracked in #3076 to be handled before launch.